packages/@webex/plugin-meetings/src/media/index.js

```diff
@@ -127,6 +127,7 @@ Media.getLocalMedia = (options, config) => {
  * @returns {MC.RoapMediaConnection}
  */
 Media.createMediaConnection = (mediaProperties, {
+  isMultistream,
```

Comment:
there is an issue with adding tests for this functionality, it requires a new release of the webrtc-media lib, so I've raised a jira:  https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-385742 to cover this and it will be done in a separate PR